packages/components/src/components/bal-list/bal-list-item-accordion-body/bal-list-item-accordion-body.tsx

```diff
@@ -28,6 +33,13 @@ export class ListItemAccordionBody {
     const inner = this.el.querySelector('.inner')
     if (inner) {
       this.contentHeight = inner.scrollHeight + 'px'
+      const parent = this.el.closest('.bal-list-item-accordion-body__parent') as HTMLElement
+      const parentHeight = parent ? parent.scrollHeight : 0
+      const parentIsThereAndIsOpen = parent && parentHeight > 0 && parent.classList.contains('is-open')
```

Comment:
Could we check for the prop or asking the component with a method. because we if the class name changes there will be no compile error